source/solvers/vof.cc

```diff
@@ -241,6 +241,8 @@ void
 VolumeOfFluid<dim>::attach_solution_to_output(DataOut<dim> &data_out)
 {
   data_out.add_data_vector(dof_handler, present_solution, "phase");
+  // Peeling/wetting output
+  data_out.add_data_vector(dof_handler, marker_pw, "marker_pw");
```

Comment:
don't we need an if to output the peeling/wetting data?

Reply:
absolutely! Well spotted, thank you